### DISCUSSION TITLE
Cloud Storage

### DISCUSSION THREAD

**Discussion by dotexe3301** [madina0801/JobTrackr#43] [Q&A]
Is there any cloud storage support to it, or any database to store the data I put in it or does it store everything on my local machine?

**Comment by mdinaya** [comment]
Hi! It saves everything locally for now, cloud storage and user registration are in plans :)

**Comment by dotexe3301** [comment]
what you are planning for that? in user registration, google oauth might help :)

**Reply by mdinaya** [reply]
I'd go with mern, I need to implement backend part firstly and I plan to use api to send applications' data to the database I think I use passport.js for user registration, if I remember correctly it has google oauth🤔 I'm not sure yet, but I think it'd be okay to make it as the main registration method

**Comment by dotexe3301** [comment]
Sounds Good :)
